projects/dev-schematics/scripts/test-dev-schematics-local.sh

```diff
@@ -0,0 +1,15 @@
+
+
+if [ $# -eq 0 ]; then
```

Comment:
I'm wonder if the version is even required?
Personally, I always have one version published to verdaccio, and I don't need to specify it when doing `npm i @spartacus/schematics`.
Not sure about other use cases, maybe other devs have different versions published?

Reply:
Exactly that's what I thought. It's a simple fallback to let dev decide on the published version.

Reply:
Okay, so I've made argument optional as we can call downgraded versions